docs/version0.37.md

```diff
@@ -33,11 +33,9 @@ The following new features and notable changes since version 0.36.x are included
 
 ### Binaries and supported environments
 
-Eclipse OpenJ9&trade; release 0.36.0 supports OpenJDK 8 and 17.
+Eclipse OpenJ9&trade; release 0.37.0 supports OpenJDK 19.
 
-Release 0.36.1 supports OpenJDK 11.
-
-Support for running OpenJDK 8 and OpenJDK 11 on CentOS 6.10 is deprecated and might be removed in a future release. OpenJ9 will not be tested with OpenJDK 11 on CentOS 6.10 after the 0.36.1 release.
```

Comment:
I don't think this should be removed so quickly.

Reply:
So, should I replace 0.36.1 with 0.37.0 in these sentences?

Reply:
I'm only referring to the CentOS 6.10 statements. Using 0.36.1 is still applicable, 0.37 is after 0.36.1, but you could write "for the 0.37 release." and keep updating it if you like.